src/Tabs.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+export const Tabs = ({ tabs, selectedTabId, selectTab }) => (
+  <>
+    {tabs.map(tab => (
+      <div key={tab.title}>
+        <button
+          type="button"
+          onClick={() => selectTab(tab.title)}
+        >
+          {tab.title}
+        </button>
+        <article>
+          {
+          selectedTabId === tab.title
+            ? tab.content
+            : null
```

Comment:
тут таокж має бути перевірка айдішніків
```suggestion
          selectedTabId === tab.id
            ? tab.content
            : null
```